targets/mghadolint/targets.go

```diff
@@ -0,0 +1,68 @@
+// Package mghadolint is a Dockerfile linter that you can employ to check for Dockerfile best
+// practices and common mistakes.
+package mghadolint
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/go-logr/logr"
+	"github.com/magefile/mage/mg"
+	"github.com/magefile/mage/sh"
+	"go.einride.tech/mage-tools/mglog"
+	"go.einride.tech/mage-tools/mgtool"
+)
+
+// version can be found here: https://github.com/hadolint/hadolint
+const version = "2.8.0"
+
+// nolint: gochecknoglobals
+var executable string
+
+func Hadolint(ctx context.Context) error {
+	ctx = logr.NewContext(ctx, mglog.Logger("hadolint"))
+	mg.CtxDeps(ctx, mg.F(prepare))
+	logr.FromContextOrDiscard(ctx).Info("running...")
+	dockerfilesRaw, err := sh.Output("git", "ls-files", "--exclude-standard", "--cached", "--others", "--", "*Dockerfile")
```

Comment:
should it be `*Dockerfile*`? (Trailing wildcard for e.g. `Dockerfile.armv5`?

Reply:
Fixed @odsod